lib/escalate.rb

```diff
@@ -72,8 +75,21 @@ def self.included(base)
 
         attr_accessor :escalate_logger_block
 
-        def escalate(exception, location_message, **context)
-          Escalate.escalate(exception, location_message, escalate_logger, **context)
+        def escalate(exception, location_message, context: {})
+          Escalate.escalate(exception, location_message, escalate_logger, context: context)
+        end
+
+        def rescue_and_escalate(location_message, context: {},
+                                exceptions: DEFAULT_RESCUE_EXCEPTIONS,
+                                pass_through_exceptions: DEFAULT_PASS_THROUGH_EXCEPTIONS,
+                                &block)
+
+          yield
+
+        rescue *Array(pass_through_exceptions)
```

Comment:
I was impressed that this works even when `pass_through_exceptions` is `[]` (or `nil`). Ruby seems fine with it.